todo_app/tests/test_pytest.py

```diff
@@ -0,0 +1,96 @@
+##from data.Item import Item
+from  todo_app.tests import inc_dec
+from todo_app import ViewModel
+from todo_app.data.Item import Item
+from datetime import date,  timedelta
+
+
+import pytest
+
+def test_increment():
+    assert inc_dec.increment(3) == 4
+
+def test_decrement():
+    assert inc_dec.decrement(3) == 2
```

Comment:
You can delete these example tests now (and the inc_dec file)

Reply:
sample tests removed
along with reference to class
class inc_dec removed form project